custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1270,6 +1270,24 @@
         "datatype": "string",
         "type": "raw",
         "property_name": "location_reassignment_last_supervisor_id"
+      },
```

Comment:
can you remove these ccs changes from this Pr. Because we may want to do these from forms as another ifa field we already have on db is coming from dp form. It wont be a good idea to mix related properties from forms and cases.

Reply:
updated!